src/nvim/main.c

```diff
@@ -229,7 +229,7 @@ int main(int argc, char **argv)
 #endif
 {
 #if defined(WIN32) && !defined(MAKE_LIB)
-  char *argv[argc];
+  char **argv = alloca((size_t)argc * sizeof(char *));
```

Comment:
According to https://github.com/neovim/neovim/wiki/Merging-patches-from-upstream-Vim#code-differences, alloca should be replace with xmalloc